hacker/admin.py

```diff
@@ -1,14 +1,39 @@
 from django.contrib import admin
 from .models import Hacker, Application, Confirmation, Team
 from django import forms
+import datetime
 
 
+''' `Hacker` '''
```

Comment:
This comment doesn't provide any additional information that can't be derived from the code. Let's ditch it